packages/Overlay/src/Backdrop.js

```diff
@@ -0,0 +1,32 @@
+import styled, { css } from "styled-components";
+import tokens from "@paprika/tokens";
+
+const openedCss = css`
+  opacity: 1;
+`;
+
+const closedCss = css`
+  opacity: 0;
+`;
+
+const states = {
+  entering: openedCss,
+  entered: openedCss,
+  exiting: closedCss,
+  exited: closedCss,
+};
+
+const Backdrop = styled.div`
```

Comment:
Can we consolidate `Backdrop.js` with `Wrapper.js` into `Overlay.styles.js`, and then use `styled.Overlay` and `styled.Backdrop` in `Overlay.js`?

Reply:
I was thinking about this. But decided to create `Backdrop` and `Wrapper` as separated components to have an ability to expose them in `index.js`. I saw your question about why I expose this components. I will answer you about this below your comment.